README.md

```diff
@@ -124,6 +124,26 @@ Use them with their validator name:
 req.checkBody('users', 'Users must be an array').isArray();
 req.checkQuery('time', 'Time must be an integer great than or equal to 5').isInt().gte(5)
 ```
+####`customSanitizers`
+_{ "sanitizerName": function(value, [additional arguments]), ... }_
+
+The `customSanitizers` option can be used to add additional sanitizers methods as needed. This option should be an `Object` defining the sanitizer names and associated functions.
+
+Define your custom sanitizers:
+
+```javascript
+app.use(expressValidator({
+ customSanitizers: {
+    toStingLowerCase: function(value) {
```

Comment:
No worries! I doesn't matter if this is multiple commits or not, so just push your new changes to this PR. If you could, ping me on here when you're finished making changes since I don't get notified when new commits are added. Thanks again!